go/vt/sqlparser/parse_test.go

```diff
@@ -559,9 +559,11 @@ var (
 	}, {
 		input: "select /* not like escape */ 1 from t where a not like b escape '$'",
 	}, {
-		input: "select /* regexp */ 1 from t where a regexp b",
+		input:  "select /* regexp */ 1 from t where a regexp b",
+		output: "select /* regexp */ 1 from t where a regexp b",
 	}, {
-		input: "select /* not regexp */ 1 from t where a not regexp b",
+		input:  "select /* not regexp */ 1 from t where a not regexp b",
+		output: "select /* not regexp */ 1 from t where a not regexp b",
```

Comment:
Same here, input and output are the same so we can drop the output and reduce the diff.